kit/cli/viper.go

```diff
@@ -272,9 +299,13 @@ func BindOptions(v *viper.Viper, cmd *cobra.Command, opts []Opt) {
 			} else {
 				IDVar(flagset, destP, o.Flag, d, o.Desc)
 			}
-			if s := v.GetString(envVar); s != "" {
-				_ = (*destP).DecodeFromString(v.GetString(envVar))
+			mustBindPFlag(v, o.Flag, flagset)
```

Comment:
Why are we adding mustBindPFlag here (what does this change?)

Reply:
This makes flags of these types get bound to viper, same as all the others. Right now flags of type `influxdb.ID` and `zapcore.Level` are being left out for no good reason.

I can revert this addition though, because one of the follow-up PRs is going to change the logic across the board for when flags are bound to viper.